hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1261,6 +1262,33 @@ public function upgrade_1036() {
     return TRUE;
   }
 
+  /**
+   * Updates the custom field to view only and disables the custom group
+   *
+   * @return bool
+   */
+  public function upgrade_1037() {
+    $customField = civicrm_api3('CustomField', 'get', [
```

Comment:
Can we make use of chained API call here?
```php
civicrm_api3('CustomField', 'get', [
  'return' => ['id']
  'name' => 'Length_Of_Service',
  'api.CustomField.create' => ['id' => '$value.id', 'is_view' => 1],
]);
```
Much neater than separating the snippet.